src/engine/Engine.ts

```diff
@@ -153,16 +153,44 @@ export interface IEngineOptions {
    backgroundColor?: Color;
 }
 
+export class EnvDefaultOptions {
```

Comment:
* Since these are `Actor` defaults, let's move these to `Actor.ts` keep the code near where what it configures

* Additionally let's not build a singleton for this, how about we hang a static bag off `Actor` instead and clone values in the `_initDefaults` method?

* Not referencing `Engine`'s prototype should also fix our current circular dependency problem

Something like
```typescript

export interface IActorDefaults {
   anchor: ex.Vector;
}

export class ActorImpl extends ...{
   public static defaults: IActorDefaults = {
      anchor: Vector.Half.clone()
   };
   ...

   _initDefaults() {
       this.anchor = Actor.defaults.anchor.clone();
   } 
}
```